tests/ErrorTests.hs

```diff
@@ -187,7 +190,7 @@ testFail19Errors =
   ["Couldn't match type ‘FLog 3 0’ with ‘CLog 3 0’"]
 
 testFail20Errors =
-  ["Couldn't match type ‘FLog 3 10’ with ‘CLog 3 10’"]
+  ["Couldn't match type ‘2’ with ’3’"]
```

Comment:
Is there a way we could keep the nicer error messages? I've been confronted with these before and it's kinda difficult to figure out where they come from.